### ISSUE TITLE
Hyperparameters for training RMU with targeted LAT

### ISSUE THREAD

**Issue by huutiendang** [aengusl/latent-adversarial-training#9]
Hi Authors,

Thank you for writing this interesting paper. I am trying to reproduce the results from section 4.3.2 RMU with targeted LAT on WMDP but I could not got results match those reported in the paper.

Here is my configs:
- forget_data_list, # Only WMDP-biology
- retain_data_list, # Wikitext
- model_name_or_path="HuggingFaceH4/zephyr-7b-beta",
- alpha=1200.0,
- layer_ids=[6,7,8],
- layer_id=8,
- param_ids=6
- lr=5e-5
- steering_coef=6.5,
- model_iterations_per_step=4,
- max_num_batches=200,
- use_pgd=True,
- pgd_layers=7,  # layers to attack
- epsilon=1,
- inner_learning_rate=5.0e-2,
- pgd_iterations_per_step=16,
- adv_loss_coefs={'toward': 1, 'away': 1},

After training, I used lm-eval-harness (v0.4.0) for evaluation, and here is the result
|      Groups      |Version|Filter|n-shot|Metric|   |Value |   |Stderr|
|------------------|------:|------|------|------|---|-----:|---|-----:|
|mmlu              |      2|none  |      |acc   |↑  |0.2601|±  |0.0037|
 |  humanities     |      2|none  |      |acc   |↑  |0.2448|±  |0.0063|
|  other          |      2|none  |      |acc   |↑  |0.2720|±  |0.0079|
|  social sciences|      2|none  |      |acc   |↑  |0.2866|±  |0.0081|
|  stem           |      2|none  |      |acc   |↑  |0.2452|±  |0.0077|
|wmdp              |      1|none  |      |acc   |↑  |0.2563|±  |0.0072|

I didn't see any discussion of the hyperparameter setting in the paper. I apologize if I overlooked any details. If possible, could you share the specific hyperparameters used for the LAT method?

Thank you.

Tien.
